climpred/classes.py

```diff
@@ -1058,8 +1058,8 @@ def _verify(
             result['lead'] = forecast['lead']
 
             if reference is not None:
-                if 'historical' in reference:
-                    hist, _ = comparison.function(hist, verif, metric=metric)
+                if 'member' in verif.dims:  # if broadcasted before
+                    verif = verif.isel(member=0)
```

Comment:
that is not a clean fix. verif is not expected to have `member`, but after comparison before initial skill computation it may have.

Reply:
Should this then be `if 'member' in verif.dims and comparison == 'm2o'`? I think thata would make it more clear why we're doing this.

Reply:
I wouldnt try to be overspecific here. that line rather fixes what shouldnt actually be. I would expect `verif` have no members.